src/ZkFold/Symbolic/Data/Bool.hs

```diff
@@ -50,6 +54,8 @@ instance Field x => BoolType (Bool x) where
 
     (||) (Bool b1) (Bool b2) = Bool $ b1 + b2 - b1 * b2
 
+    xor (Bool b1) (Bool b2) = Bool $ b1 + b2 - (b1 * b2 + b1 * b2)
```

Comment:
In #65, we aim to make constants free. In particular, that would mean that `(b1 * b2 + b1 * b2) == 2 * b1 * b2`. So, the whole `xor` function can be implemented with one Plonk constraint.

Reply:
But since it is not implemented yet, maybe we could leave it like that.